test/worker.test.js

```
@@ -0,0 +1,151 @@
+// Mock Cloudflare environment
+const mockEnv = {
+  RATE_LIMIT_KV: {
+    get: jest.fn(),
+    put: jest.fn().mockResolvedValue(undefined),
+  },
+  API_PROBE_TOKEN: 'test-token',
+  waitUntil: jest.fn()
+};
+
+// Mock global objects
+const mockRequest = new Request('https://example.com', {
```

ESLint

[error] 'mockRequest' is assigned a value but never used.

(no-unused-vars)